src/Engine/EmitMSIL/CodeGenIL.cs

```diff
@@ -115,7 +119,7 @@ public CodeGenIL(IDictionary<string, IList> input, string filePath)
             // 4. Create method ("Execute"), get ILGenerator 
             var execMethod = BuilderHelper.CreateMethod(type, "Execute",
                 System.Reflection.MethodAttributes.Static, typeof(void), new[] { typeof(IDictionary<string, IList>),
-                typeof(IDictionary<int, IEnumerable<MethodBase>>), typeof(IDictionary<string, object>)});
+                typeof(IDictionary<int, IEnumerable<ProtoCore.CLRFunctionEndPoint>>), typeof(IDictionary<string, object>), typeof(ProtoCore.MSILRuntimeCore)});
```

Comment:
Can we just add a `using ProtoCore` to avoid prefixing the namespace everywhere and make it less verbose?

Reply:
that would cause a lot of conflicts with Type (System.Type and ProtoCore.Type, and others I think)